src/main/compa/models/Token.java

```diff
@@ -0,0 +1,34 @@
+package main.compa.models;
+
+import com.google.gson.annotations.Expose;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.bson.types.ObjectId;
+import org.mongodb.morphia.annotations.*;
+
+import java.util.Date;
+
+@Entity("token")
+@Indexes({
+})
+public class Token {
```

Comment:
seems excessive to not only store outdated tokens (arrayList inside user???) , but to make it an object with an id, and a beginning date. (could be useful to store an expiryDate however). why not just a String and an expiry date, inside the user?

Reply:
Right